index.html

```diff
@@ -1,16 +1,121 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
+
+  <title>Great Idea!</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/styles.css">
 
```

Comment:
You are missing the meta tag `content="width=device-width, initial-scale=1"`

Reply:
Can you explain what does it do?

Reply:
From what I understood, it controls the window in which the content is seen. I believe that manipulating the `initial-scale` creates a sort of zoom-in or zoom-out effect! I must have omitted it in the submission, so will update